README.md

```diff
@@ -71,6 +71,17 @@ gradle build
 gradle run -DmainClass=com.google.swarm.tokenization.DLPTextToBigQueryStreamingV2 -Pargs="--region=<region> --project=<project_id> --streaming --enableStreamingEngine --tempLocation=gs://<bucket>/temp --numWorkers=1 --maxNumWorkers=2 --runner=DataflowRunner --filePattern=s3://<bucket>>/file.csv --dataset=<name>  --inspectTemplateName=<inspect_template> --deidentifyTemplateName=<deid_tmplate> --awsRegion=<aws_region> --awsCredentialsProvider=$AWS_CRED"
 ```
 
+## DeIdentification of jsonl files
+JSONL file format uses newline characters to separate JSON values. A sample file can be found at /src/test/resources/sample_data.jsonl
```

Comment:
How does the pipeline react if the line is a JSON array? If we only support having valid JSON Object per line, may be we should word this as:

The pipeline supports JSONL file format where each line is a valid JSON Object and newline character is used to separate JSON objects.

Please check if we can have a link to the sample file:
A sample file can be found in [test resources](/src/test/resources/sample_data.jsonl)